src/components/Error.js

```diff
@@ -1,14 +1,24 @@
-import styled from 'styled-components'
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
 
-function Error({ err }) {
-    return (
+function ErrorEl({ err }) {
```

Comment:
What does `ErrorEl` stands for? Maybe makes sense to come up with more meaningful name because `El` part can be confusing.

Reply:
Fixed that